lib/core/datatable/components/datatable/date-cell.component.ts

```diff
@@ -50,8 +51,8 @@ export class DateCellComponent extends DataTableCellComponent {
         return 'medium';
     }
 
-    constructor(userPreferenceService: UserPreferencesService) {
-        super();
+    constructor(userPreferenceService: UserPreferencesService, apiService: AlfrescoApiService) {
```

Comment:
would it better as we discuss rename apiService to alfrescoApiService? it helps consistency

Reply:
updated name